master/getting-started/bare-metal/bare-metal.md

```diff
@@ -559,7 +590,10 @@ out of a host interface:
 
 For packets that are sent from a local workload out of a host interface:
 
-- No host endpoint policies apply.
+- No untracked or pre-DNAT host endpoint policies apply.
+
+- Normal policies apply if `applyOnForward` is `true`, specifically the egress rules of the normal policies that
+apply to the outgoing interface.  (The reverse direction is allowed by conntrack state.) Normal policies do not apply if `applyOnForward` is `false`.
```

Comment:
Please, only one space between sentences....so, "interface.  (The" --> "interface. (The"